src/Service.GraphQLBuilder/Mutations/CreateMutationBuilder.cs

```diff
@@ -280,6 +367,39 @@ private static NameNode GenerateInputTypeName(string typeName)
                 new NamedTypeNode(returnEntityName),
                 fieldDefinitionNodeDirectives
             );
+
+            // Multiple insertion node.
+            FieldDefinitionNode createMultipleNode = new(
+                location: null,
+                new NameNode($"create{GetInsertMultipleMutationName(singularName, GetDefinedPluralName(name.Value, entity))}"),
+                new StringValueNode($"Creates multiple new {singularName}"),
+                new List<InputValueDefinitionNode> {
+                new(
+                    location : null,
+                    new NameNode(ARRAY_INPUT_ARGUMENT_NAME),
+                    new StringValueNode($"Input representing all the fields for creating {name}"),
+                    new ListTypeNode(new NonNullTypeNode(new NamedTypeNode(input.Name))),
+                    defaultValue: null,
+                    new List<DirectiveNode>())
+                },
+                new NamedTypeNode(QueryBuilder.GeneratePaginationTypeName(GetDefinedSingularName(dbEntityName, entity))),
```

Comment:
can you use named params here too? on line 374 you use the `GetDefinedPluralName` but on 385 you used `GetDefinedSingularName` its unclear what the purpose of the usage of singular/plural is